src/components/Form/Example.md

```diff
@@ -160,3 +160,74 @@ const { Form, Button, TextField } = require('precise-ui');
   </div>
 </Form>
 ```
+
+Managed `Form` with default value and reset button
+
+```jsx
+const { Form, Button, TextField, DropdownField } = require('precise-ui');
+
+const filterDefinitions = {
+  timeRanges: {
+    label: 'Time Range',
+    name: 'timeRanges',
+    data: [
+      { content: 'Last 7 days', key: '7' },
+      { content: 'Last 30 days', key: '30' },
+      { content: 'Last 60 days', key: '60' },
+      { content: 'Last 90 days', key: '90' },
+      { content: 'Last year', key: '365' },
+    ],
+  },
+  types: {
+    label: 'Types',
+    name: 'types',
+    multiple: true,
+    data: [
+      { content: 'Statement', key: 'Statement' },
+      { content: 'Invoice', key: 'Invoice' },
+      { content: 'Credit', key: 'Credit' },
+    ],
+  },
+  statis: {
+    label: 'Status',
+    name: 'statis',
```

Comment:
typo? Maybe `status`?

Reply:
actually I thought about the plural version of "status" (at least in German it is called like this ;))
but sure, your right, maybe it is better to switch to "status" instead!